src/gen_experiments/utils.py

```diff
@@ -135,6 +135,75 @@ def _alert_short(arr):
     x_dot_test = [xi for xi in x_dot_test]
     return dt, t_train, x_train, x_test, x_dot_test, x_train_true
 
+def gen_pde_data(
+    rhs_func,
+    init_cond,
+    args,
+    seed=None,
+    noise_abs=None,
+    noise_rel=None,
+    nonnegative=False,
+    dt=0.01,
+    t_end=10,
+):
+    """Generate random training and test data
+
+    Note that test data has no noise.
```

Comment:
1. Mention all the simplifications from `gen_data()` (e.g. the removal of `n_trajectories` and randomized starting points to ICs.  And for the same reason, training=test)
2. Add `dt` and `t_end` to docstring
3. Remove nonnegative arg (since it isn't used)
4. remove types from docstring and add them to function signature (e.g. the string `rhs_func (Callable): ...` becomes `rhs_func: ...` and the code `rhs_func,\n` becomes `rhs_func: Callable,\n`.  Import from `typing` where necessary
5. When you do that for `init_cond`, subtype `List` to `List[np.ndarray]`.  In docstring, explain the required shape.
6. Be more descriptive on the first line of the docstring, e.g. "Generate random PDE training and test data"

Reply:
This is in the main branch, I made the changes but they are in the pdes branch,  should I merege these two branches before commiting?

Reply:
Copy the changes into `yb6599:main` then merge this branch back into `yb6599:pdes`.  That way the changes in that PR don't appear here, and you can resolve the merge conflicts on your computer.